packages/typir/src/typir.ts

```diff
@@ -73,11 +73,14 @@ export const DefaultTypirServiceModule: Module<TypirServices> = {
     validation: {
         collector: (services) => new DefaultValidationCollector(services),
         constraints: (services) => new DefaultValidationConstraints(services),
-    }
+    },
 };
 
-export function createTypirServices(customization: Module<TypirServices, PartialTypirServices> = {}): TypirServices {
-    return inject(DefaultTypirServiceModule, customization);
+export function createTypirServices(
+    customization1: Module<TypirServices, PartialTypirServices> = {},
```

Comment:
What are these two customizations good for?

Reply:
The function `createTypirServices` can be used to start Typir in general. (This is not used in case of Langium.)

The two optional arguments should be used to customize the Typir default module, e.g. for LOX the output of this function should be passed:

```typescript
export function createLoxTypirModule(langiumServices: LangiumSharedServices): Module<LangiumServicesForTypirBinding, PartialTypirLangiumServices> {
    return {
        // specific configurations for LOX
        TypeCreator: (typirServices) => new LoxTypeCreator(typirServices, langiumServices),
    };
}
```